user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,65 @@
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest<!-- (see Update Strapi version or refer to our migration guides to update your Strapi application) -->. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content managed with Strapi itself, are ruled by a system of permissions. From your Strapi admin panel, you have the possibility to define, at a detailed level, the roles and permissions of all administrators and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
+
+With all this in mind, you should be ready to start your Strapi experience!
+
+
+## Accessing the admin panel
+
+The admin panel is the back office of your Strapi application. From the admin panel, you will be able to manage content types, and write their actual content. It is also from the admin panel that you will manage users, both administrators and end-users of your Strapi application.
+
+::: tip IMPORTANT
+In order to access the admin panel, your Strapi application must be launched<!-- (see Installing Strapi) -->, and you must be aware of its URL.
+:::
+
+![User profile](../assets/getting-started/login-page.png)
+
+To access the admin panel:
+
+1. Go to the URL of your Strapi application.
+2. (optional) Click on the **Open the administration** button.
+3. Enter your credentials to log in.
+4. Click on the **Log in** button. You should be redirected to the homepage of the admin panel.
+
+
+## Setting up your user profile
+
+If you are a new administrator, we recommend making sure your user profile is all set, before diving into your Strapi application. From your admin panel user profile, you are able to modify your user information, such as name, username, email or password.
+
+![User profile](../assets/getting-started/user-information-profile.png)
+
+To modify your user information:
+
+1. Click on your user name in the top right hand corner of the admin panel.
+2. In the drop-down menu, click on **Profile**.
+3. Modify the information of your choice:
+
+| User information       | Instructions                                                                                             |
```

Comment:
Doesn't matter here but I believe VSCode has an extension that will automatically format these tables in a clean way, regardless once vuepress parses the file it's the same.

Don't know if "cleanliness" applies in documentation for stuff like this, lord knows the rest of our old docs are not clean :see_no_evil:

Reply:
Oh I didn't know that! I used a MarkDown table generator to help me because MarkDown tables are quite annoying to write 🙈  I'll take a look at VSCode extensions.

Reply:
They are extremely difficult and time consuming to write lol